mautic/mautic
# [DPMMA-2341] Missing variables in preference center token templates

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [✅]
| New feature/enhancement? (use the a.x branch)      | [❌]
| Deprecations?                          | [❌]
| BC breaks? (use the c.x branch)        | [❌]
| Automated tests included?              | [❌] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | #12894  



#### Description:

This PR adds conditions that check whether the variables used in the templates channelfrequency and preferredchannel are correctly defined and can be displayed.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Follow the steps from "How can we reproduce this issue" #12894 Error should no longer appear.
